Projects/hCoreUI/Sources/RemoteVectorIcon.swift

```diff
@@ -45,8 +45,6 @@ extension RemoteVectorIcon: Viewable {
         func renderPdfDocument(pdfDocument: CGPDFDocument) {
             let imageViewSize = imageView.frame.size
 
-            if let image = imageView.image { if image.size == imageViewSize { return } }
```

Comment:
Maybe we should check if traits actually changed? Not sure this will be super performant.

Reply:
This `renderPdfDocument` was being called when traits/layout is changed. I think it should not affect the performance too much in that case 🤔

Reply:
layout happens quite often actually 😓

Reply:
We actually do a check here: https://github.com/HedvigInsurance/ugglan/blob/0961647812451371a404266b98772a7ca87b5df3/Projects/hCoreUI/Sources/RemoteVectorIcon.swift#L86

So I guess its fine to remove that line :)